jablib/src/test/java/org/jabref/logic/layout/LayoutEntryTest.java

```diff
@@ -73,32 +73,38 @@ public String layout(String layoutFile, BibEntry entry) throws IOException {
         return layout.doLayout(entry, null);
     }
 
-    @Test
-    void parseMethodCalls() {
-        assertEquals(1, LayoutEntry.parseMethodsCalls("bla").size());
-        assertEquals("bla", LayoutEntry.parseMethodsCalls("bla").getFirst().getFirst());
-
-        assertEquals(1, LayoutEntry.parseMethodsCalls("bla,").size());
-        assertEquals("bla", LayoutEntry.parseMethodsCalls("bla,").getFirst().getFirst());
-
-        assertEquals(1, LayoutEntry.parseMethodsCalls("_bla.bla.blub,").size());
-        assertEquals("_bla.bla.blub", LayoutEntry.parseMethodsCalls("_bla.bla.blub,").getFirst().getFirst());
-
-        assertEquals(2, LayoutEntry.parseMethodsCalls("bla,foo").size());
-        assertEquals("bla", LayoutEntry.parseMethodsCalls("bla,foo").getFirst().getFirst());
-        assertEquals("foo", LayoutEntry.parseMethodsCalls("bla,foo").get(1).getFirst());
+    @ParameterizedTest
+    @CsvSource(delimiterString = "->", textBlock = """
```

Comment:
We like the default separator"," more 😅

Moreover with `->` the intention "expected" and "input" is wrong.